.circleci/config.yml

```diff
@@ -20,6 +20,28 @@ parameters:
     type: integer
     default: 5
 
+commands:
+  simulate_nested:
+    description: "Runs simulations of a nested task"
+    parameters:
+      task:
+        type: string
+    steps:
+      - checkout
+      - run:
+          name: "simulate nested << parameters.task >>"
+          command: |
+            just install
+            cd tasks/<< parameters.task >>
+            SIMULATE_WITHOUT_LEDGER=true just \
+              --dotenv-path $(pwd)/.env \
+              --justfile ../../../nested.just \
+              simulate foundation
+            SIMULATE_WITHOUT_LEDGER=true just \
+              --dotenv-path $(pwd)/.env \
+              --justfile ../../../nested.just \
+              simulate council
```

Comment:
we should leave this one in for future tasks, which can then just run this command instead of copy-pasting the simulation invocation, see below `simulate_sep_025`

Reply:
nice, I like this